ios/FluentUI/Navigation/Shy Header/ShyHeaderController.swift

```diff
@@ -523,10 +519,10 @@ class ShyHeaderController: UIViewController {
     }
 
     /// Updates based on the current navigation bar style.
-    private func updateNavigationBarStyle() {
+    public func updateNavigationBarStyle(theme: FluentTheme) {
```

Comment:
why is this public? shyheader isn't a public class?

Reply:
Doesn't need to be public, I'll fix it!